src/corona_hakab_model/infection.py

```diff
@@ -36,8 +36,9 @@ def _perform_infection(self) -> List[Agent]:
         # v = [True if an agent can infect other agents in this time step]
         v = np.random.random(len(self.manager.agents)) < self.manager.contagiousness_vector
 
+        # TODO save manifests in a queue-ish data structure
         # u = mat dot_product v (log of the probability that an agent will get infected)
-        u = self.manager.matrix.prob_any(v)
+        manifests, u = self.manager.matrix.POA(v)
```

Comment:
rename manifests to `_` to show that it isn't being used

Reply:
done.